torch_xla/csrc/xla_sharding_util.cpp

```diff
@@ -385,7 +386,7 @@ std::vector<std::vector<at::indexing::TensorIndex>>
 ShardingUtil::GetShardIndicesForDevices(
```

Comment:
I'm thinking we should leave this function untouched and add a new helper function to get indices for minibatch sharding. Wdyt?